features/policies.feature

```diff
@@ -27,11 +30,13 @@ Scenario: Creating a policy programme that is part of another
   And a policy called "CO2 reduction" is indexed for search
   And a policy called "Global warming" is republished to publishing API
 
+@future-policies
 Scenario: Associating a policy with an organisation
   Given a policy exists called "Global warming"
   When I associate the policy with an organisation
   Then the policy should be linked to the organisation when published to publishing API
 
+@future-policies
```

Comment:
You can tag the entire feature if needed: https://github.com/cucumber/cucumber/wiki/Tags